README.md

```diff
@@ -1,5 +1,8 @@
 #  AI-lab Software Templates
 
+> [!IMPORTANT]  
+> The software templates in this repository require OpenShift Pipelines and OpenShift GitOps installed, alongside their respective plugins in Red Hat Developer Hub. For more information on configuring them, please see [the installer we have prepared](https://github.com/redhat-ai-dev/ai-rhdh-installer/).
```

Comment:
Tiny nit, it's totally ok for me with the extisting content. I only think better to use the `ai-rhdh-installer` "branding".

```suggestion
> The software templates in this repository require OpenShift Pipelines and OpenShift GitOps installed, alongside their respective plugins in Red Hat Developer Hub. For more information on configuring them, please see the [ai-rhdh-installer](https://github.com/redhat-ai-dev/ai-rhdh-installer/).
```

Reply:
Updated.